internal/databroker/config_source.go

```diff
@@ -84,40 +89,50 @@ func (src *ConfigSource) rebuild(firstTime bool) {
 	var additionalPolicies []config.Policy
 
 	// add all the config policies to the list
-	for _, cfgpb := range src.dbConfigs {
+	for id, cfgpb := range src.dbConfigs {
 		cfg.Options.ApplySettings(cfgpb.Settings)
+		var errCount uint64
```

Comment:
nit: `errCount` is now a ways away from its use. We might want to move this to just above the for loop